deployment_report/model/viya_deployment_report.py

```diff
@@ -109,6 +109,22 @@ def gather_details(self, kubectl: KubectlInterface,
         # create dictionary to store gathered resources #
         gathered_resources: Dict = dict()
 
+        # start by gathering details about ConfigMap #
+        cadence_info: Optional[Text] = None
+        try:
+            ViyaDeploymentReportUtils.gather_resource_details(kubectl, gathered_resources, api_resources,
+                                                              k8s_kinds.CONFIGMAP)
+            for item in gathered_resources[k8s_kinds.CONFIGMAP]['items']:
```

Comment:
the logic to gather cadence info should be relocated to the function get_cadence_data

Reply:
updated with new code.